lte/gateway/c/oai/include/sctp_messages_types.h

```diff
@@ -52,12 +54,16 @@
 #define SCTP_CLOSE_ASSOCIATION(msg) (msg)->ittiMsg.sctp_close_association
 #define SCTP_MME_SERVER_INITIALIZED(msg)                                       \
   (msg)->ittiMsg.sctp_mme_server_initialized
+#define SCTP_AMF_SERVER_INITIALIZED(msg)                                       \
+  (msg)->ittiMsg.sctp_amf_server_initialized
 
 typedef struct sctp_data_cnf_s {
   bstring payload;
   sctp_assoc_id_t assoc_id;
   sctp_stream_id_t stream;
   uint32_t mme_ue_s1ap_id;
+  uint32_t amf_ue_ngap_id;
```

Comment:
it would be better to merge these into `agw_ue_xap_id` and put a comment next to it saying this will be set to mme_ue_s1ap_id or amf_ue_ngap_id based on the ap_id.

Reply:
okay